json-schemas/src/account-stats.json

```diff
@@ -1097,6 +1094,36 @@
           "inclusiveMinimum": 0,
           "description": "Total number of messages on channels where delta generation was attempted but where a delta was not successfully generated, so Ably just broadcasted the original, see https://ably.com/docs/channels/options/deltas."
         },
+        "messages.processed.batching.simple.inboundMessages": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of messages going through the batching pipeline with the 'simple' batching policy."
```

Comment:
wondering if "going through the batching pipeline" might be a little bit too ably-internal-jargon. maybe just "aggregated into batches" (or "aggregated into batches before conflation" for the conflation type)? or "published on channels with batching enabled"?